content/source/docs/extend/best-practices/index.html.md

```diff
@@ -46,5 +54,6 @@ Terraform would expect a resource to behave.
 
 [Click here to learn more.][drift]
```

Comment:
can you delete this line for me? I don't think its needed and it's confusingly placed.

Reply:
I need to add a synopsis for the versioning page -- let me address this as well in that upcoming PR after this one. 👍